src/aave-npm-packages.yaml

```
@@ -7,35 +7,415 @@ created: "2022-08-02T04:15:13.911Z"
 categories:
   - developer
   - engineering
-thumbnail: https://d31h13bdjwgzxs.cloudfront.net/guides/eth/aave/AAVE_NPM_Packages.png
+thumbnail: https://d31h13bdjwgzxs.cloudfront.net/guides/eth/aave/Borrowing_in_aave.png
 publishStatus: Live
 socialShareImage:
 showIncorrectOnCompletion: true
 steps:
-  - uuid: 18f145d1-e807-4da3-98f3-57d010354ce0
-    name: List of packages
+  - uuid: 5db956ef-4e00-4ac1-8c25-64c5dbebfa73
+    name: Aave core and stake
     content: |
-      List of packages
-      See here - https://www.npmjs.com/search?q=%40aave
+
+     ## Aave coreV3
+     This package contains the smart contracts source code and markets configuration related to Aave Protocol V3. 
+     It uses Docker Compose and Hardhat as development environment for compilation, testing and deployment tasks.
+
+     You can install @aave/aave-v3-core as an npm package to your hardhat or truffle workspace to get access to contracts and 
+     interfaces provided by Aave V3
+
+     The repository uses Docker Compose to manage sensitive keys and load the configuration. 
+     Before any action like test or deploy, you must run docker-compose up to start the contracts-env container, and then connect to the container console through docker-compose exec contracts-env bash.
+     You can run a full test suite using Docker Compose commands.
```

````suggestion

You can install `@aave/core-v3` as an NPM package in your Hardhat or Truffle project to import the contracts and interfaces:

`npm install @aave/core-v3`

Import at Solidity files:

```
import {IPool} from "@aave/core-v3/contracts/interfaces/IPool.sol";

contract Misc {

  function supply(address pool, address token, address user, uint256 amount) public {
    IPool(pool).supply(token, amount, user, 0);
    {...}
  }
}
```

The JSON artifacts with the ABI and Bytecode are also included into the bundled NPM package at `artifacts/` directory.

Import JSON file via Node JS `require`:

```
const PoolV3Artifact = require('@aave/core-v3/artifacts/contracts/protocol/pool/Pool.sol/Pool.json');

// Log the ABI into console
console.log(PoolV3Artifact.abi)
```
````